### PR TITLE
#167164940 resend verification email endpoint

### PR DESCRIPTION
**Pull request by WilliamsOhworuka**
#### Pivotal Tracker Story

[#167164940](https://www.pivotaltracker.com/story/show/167164940)

#### What does this PR do?
provides resend verification email functionality

#### Summary of Task
create endpoint to resend verification email
 
#### How can this be tested?
- git clone branch `feature/167164940/resend-verification-email-endpoint`
- run npm install
- start dev server npm run dev:start

#### Screenshots (if appropriate)

Screenshot 2019-09-06 at 4 03 26 PM

### CODE REVIEW

File: src/controllers/AuthController.js [hunk 1]
```diff
@@ -176,10 +177,39 @@ const resetPassword = async (request, response) => {
...
+    const { email, firstName, isVerified } = user;
+    if (isVerified) {
+      return responseMessage(res, 403, {
+        error: 'you\'re already verified',
+      });
+    }
+
+    await verifyUser({ id, email, firstName });
+    return responseMessage(res, 200, {
+      message: 'request sent you\'ll a mail shortly',
```
Comment (Nerocodes) [inline src/controllers/AuthController.js]: I think there's a typo here. "you'll a mail shortly"

File: src/controllers/AuthController.js [hunk 2]
```diff
@@ -176,10 +177,39 @@ const resetPassword = async (request, response) => {
...
+    if (!user) {
+      return responseMessage(res, 404, {
+        error: 'User not found',
+      });
+    }
+
+    const { email, firstName, isVerified } = user;
+    if (isVerified) {
+      return responseMessage(res, 403, {
+        error: 'you\'re already verified',
```
Comment (Nkemjiks) [inline src/controllers/AuthController.js]:
```suggestion
        error: 'You are already verified',
```

File: src/controllers/AuthController.js (hunk 1 above)
Comment (Nkemjiks) [inline src/controllers/AuthController.js]:
```suggestion
      message: 'Request sent. You will receive an email shortly',
```

**Review by Nkemjiks — CHANGES_REQUESTED** [review]

File: src/controllers/AuthController.js (hunk 2 above)
Comment (WilliamsOhworuka) [reply src/controllers/AuthController.js]: done

File: src/controllers/AuthController.js (hunk 1 above)
Comment (WilliamsOhworuka) [reply src/controllers/AuthController.js]: done

Comment (WilliamsOhworuka) [reply src/controllers/AuthController.js]: done

**Review by Nerocodes — APPROVED** [review]

**Review by Nkemjiks — APPROVED** [review]

**Pull request merged**